sros2/sros2/verb/generate_artifacts.py

```diff
@@ -31,11 +31,13 @@ class GenerateArtifactsVerb(VerbExtension):
     """Generate keys and permission files from a list of identities and policy files."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('-k', '--keystore-root-path', help='root path of keystore')
+        arg = parser.add_argument(
+            '-k', '--keystore-root-path',
+            help='root path of keystore')
         arg.completer = DirectoriesCompleter()
         parser.add_argument(
-            '-n', '--node-names', nargs='*', default=[],
-            help='list of identities, aka ROS node names')
+            '-c', '--contexts', nargs='*', default=[],
```

Comment:
`-n` could also be kept. It should be interpreted as "names" (i.e.: "security contexts names").

Reply:
Context paths do not have an equivalent meaning as node names, and I think the args should conflate the two to make that distinction clear, so the use recognizes foxy has shifted to context.

Reply:
Sounds reasonable, applies to https://github.com/ros2/sros2/pull/177#discussion_r394644744 too.